sentry/src/main/java/io/sentry/protocol/User.java

```diff
@@ -150,19 +150,43 @@ public void setIpAddress(final @Nullable String ipAddress) {
   /**
    * Gets other user related data.
    *
+   * @deprecated use getData instead
```

Comment:
You can use the @link javadocs
Eg `{{@link SentryOptions#getShutdownTimeoutMillis()} }`

Reply:
updated